### PR TITLE
Materialized Snackbar with better positioning

### PR DESCRIPTION
**Pull request by samatagrav**
The popup did not position correctly when scrolling the page so I had to modify it.

### CODE REVIEW

File: lib/src/components/snackbar/snackbar.dart [hunk 1]
```diff
@@ -86,21 +90,40 @@ class SkawaSnackbarComponent implements OnDestroy {
...
   bool displayPopup = false;
+  bool isFixed = false;
 
   bool get useAction => actionLabel != null;
   bool showAnimation = false;
 
+  DivElement parentDiv;
+  bool first = true;
+
   void visibleChanged(bool visible) {
```
[the remark this answers is not in the document]
Comment (samatagrav) [reply lib/src/components/snackbar/snackbar.dart]: removed unnecessary parameter.

File: lib/src/components/snackbar/snackbar.dart [hunk 2]
```diff
@@ -86,21 +90,40 @@ class SkawaSnackbarComponent implements OnDestroy {
...
+    if (first) {
+      parentDiv = document.body.querySelector('div[pane-id="${popupComponent.resolvedPopupRef.uniqueId}"]');
+      _disposer.addStreamSubscription(popupComponent.onOpened.listen((_) {
+        parentDiv.style.setProperty("transform", "translateX(30px) translateY(${window.screen.height - 130}px)");
```
[the remark this answers is not in the document]
Comment (samatagrav) [reply lib/src/components/snackbar/snackbar.dart]: changed to 180

File: lib/src/components/snackbar/snackbar.html [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-       [source]="documentSource"
-       [offsetX]="24"
-       [offsetY]="-24"
-       (visible)="visibleChanged($event)"
-       ink
-       slide="x">
+<material-popup [source]="documentSource"
+                [offsetX]="24"
+                [offsetY]="-24"
+                (open)="visibleChanged($event)"
```
[the remark this answers is not in the document]
Comment (samatagrav) [reply lib/src/components/snackbar/snackbar.html]: removed

File: lib/src/components/snackbar/snackbar.html [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
+                (close)="visibleChanged($event)"
```
[the remark this answers is not in the document]
Comment (samatagrav) [reply lib/src/components/snackbar/snackbar.html]: removed

**Comment by valakis** [comment]
LGTM

**Comment by daniel-v** [comment]
thanks people

**Pull request merged**